modules/Bio/Otter/ServerAction/Region.pm

```diff
@@ -713,8 +713,8 @@ sub fill_exon_align_evidence() {
         if ($exon_toplevel) {
           $exon = $exon_toplevel;
           if ($start_Exon) {
-            $transcript->translation->start_Exon($exon_toplevel) if ($start_Exon == $exon);
-            $transcript->translation->end_Exon($exon_toplevel) if ($end_Exon == $exon);
+            $transcript->translation->start_Exon($exon) if ($start_Exon == $exon);
+            $transcript->translation->end_Exon($exon) if ($end_Exon == $exon);
           }
```

Comment:
Because you set $exon to $exon_toplevel, this if statement is probably not needed